json-rpc/docs/type_event.md

```diff
@@ -148,9 +148,11 @@ Event emitted after minted, destination address received the minted coins.
 
 Event emitted when a new account is created
 
-| Name    | Type   | Description                    |
-|---------|--------|--------------------------------|
-| type    | string | Constant string "createaccount"|
+| Name            | Type   | Description                    |
+|-----------------|--------|--------------------------------|
+| type            | string | Constant string "createaccount"|
+| created_address | string | The address created on-chain   |
+| role_id         | u64    | Role id, see [Move resource RoleId](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Roles.md#resource-roleid) for more details |
```

Comment:
Could additionally link to LIP2 here: https://github.com/libra/lip/blob/master/lips/lip-2.md#move-implementation